tutor/specs/e2e/course-settings.e2e.ts

```diff
@@ -27,7 +27,8 @@ describe('Course Settings', () => {
             await expect(page).toHaveSelector('.course-detail-settings-form')
             const courseName = page.$eval('input#course-name', (el: any) => el.value)
             await page.click('testEl=delete-course-btn')
-            await expect(page).not.toHaveSelector('testEl=disabled-delete-course-message', { timeout: 100 })
+            await expect(page).toHaveText('testEl=delete-course-message', 'delete', { timeout: 100 })
```

Comment:
we don't need to use use `timeout` option on a `toHaveXX` expectation.  it's only needed on a `not.toHaveXX` one.  If we do not specify it on the `not` then jest will wait until it timeouts just in case it becomes true at some point